src/source/Ice/IceAgent.c

```diff
@@ -1280,8 +1299,11 @@ STATUS iceAgentSendSrflxCandidateRequest(PIceAgent pIceAgent)
                     if (pIceServer->ipAddress.family == pCandidate->ipAddress.family) {
                         transactionIdStoreInsert(pIceAgent->pStunBindingRequestTransactionIdStore,
                                                  pBindingRequest->header.transactionId);
+                        checkSum = COMPUTE_CRC32(pBindingRequest->header.transactionId, ARRAY_SIZE(pBindingRequest->header.transactionId));
+                        pIceAgent->rtcIceServerDiagnostics[pCandidate->iceServerIndex].totalRequestsSent++;
                         CHK_STATUS(iceAgentSendStunPacket(pBindingRequest, NULL, 0, pIceAgent, pCandidate,
                                                           &pIceServer->ipAddress));
+                        CHK_STATUS(hashTableUpsert(pIceAgent->rtcIceServerDiagnostics[pCandidate->iceServerIndex].requestTimestampDiagnostics, checkSum, GETTIME()));
```

Comment:
need to also increment totalRequestsSent when send binding request from turn candidates.

Reply:
Fixed it by checking for candidate type relay and then updating the requests count. Also capturing request sent timestamp in `iceCandidatePairCheckConnection`